features/docker.feature

```diff
@@ -0,0 +1,69 @@
+@uses.config.contract_token
+Feature: Build docker images with ua services
+
+    @slow
+    @docker
+    @series.focal
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Build docker images with ua services
+        Given a `focal` machine with ubuntu-advantage-tools installed
+        When I have the `<container_release>` debs under test in `/home/ubuntu`
+        When I run `apt-get install -y docker.io` with sudo
+        When I create the file `/home/ubuntu/Dockerfile` with the following:
+        """
+        FROM ubuntu:<container_release>
+
+        COPY ./ubuntu-advantage-tools.deb /ua.deb
+
+        RUN --mount=type=secret,id=ua-attach-config \
+            apt-get update \
+            && apt-get install --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
+
+            && dpkg -i /ua.deb \
+
+            && ua attach --attach-config /run/secrets/ua-attach-config \
+
+            # Normally an apt upgrade is recommended, but we dont do that here
+            # in order to measure the image size bloat from just the enablement
+            # process
+            # && apt-get upgrade -y \
+
+            && apt-get install -y <test_package_name> \
+
+            # If you need ca-certificates, remove it from this line
+            && apt-get purge --auto-remove -y ubuntu-advantage-tools ca-certificates \
```

Comment:
No but there's no reason to because it is already in a previous layer, so removing won't affect the size. And we're compensating for the deb file in the size comparison test

I suppose we could use a squashing tool like you suggest above, but I don't want to deviate too much from what we expect a customer would do